app/models/councillor.rb

```diff
@@ -6,6 +6,12 @@ class Councillor < ActiveRecord::Base
   validates :authority, :email, presence: true
   validates :image_url, format: { with: /\Ahttps/, message: "must be HTTPS" }, allow_blank: true
 
+  def initials
```

Comment:
I'm going to leave this as is. I think it's clear from the tests and implementation what it does and if it get's used elsewhere and needs to be made more flexible we can change it then.